persistent/persistence.py

```diff
@@ -230,16 +252,16 @@ def _get_status(self):
     def __getattribute__(self, name):
         """ See IPersistent.
         """
-        if (not name.startswith('_Persistent__') and
```

Comment:
We're not. The private `_Persistent__` names are in the `_SPECIAL_NAMES` `set`. It turns out (according to jitviewer) PyPy is really good at inlining the assembly code to do *everything* necessary to implement `name in _SPECIAL_NAMES`, but the string comparison still involves a function call.

```python
       if (not name.startswith('_p_') and
            name not in _SPECIAL_NAMES):
            if oga(self, '_Persistent__flags') is None:
                oga(self, '_p_activate')()
            oga(self, '_p_accessed')()
        return oga(self, name)
```